gcp-serverless/index.js

```diff
@@ -3,16 +3,28 @@ const config = require('./config.json')
 const { getColorCodes } = require('./getColorCodes')
 const callParticleApi = require('./callParticleApi')
 
-exports.callparticle = (request, response) => {
-  // @todo, does this var need to be escaped?
-  const deviceId = request.query.coreid
-  const particleToken = config.PARTICLE_TOKEN
-  const makeRequest = async () => {
-    const colorList = await getColorCodes(config.GITHUB_TOKEN)
-    await callParticleApi(particleToken, deviceId, colorList)
-    response.status(200).send(colorList)
+const makeRequest = async (coreid) => {
+  const colorList = await getColorCodes(config.GITHUB_TOKEN)
+
+  const configObject = {
+    'colorList': colorList,
+    // @todo, does this var need to be escaped?
```

Comment:
Do you plan to resolve this todo prior to merging?

Reply:
No, this `todo`, was present before this PR and the question remains open.